From 571c141b434dff13494c6a3afe621f63a8e610e9 Mon Sep 17 00:00:00 2001
From: Andrey Zhizhikin <andrey.z@gmail.com>
Date: Mon, 27 Jan 2020 14:29:34 +0000
Subject: [PATCH] team_basic_test.py: use python3 interpreter

Use python3 since python2 is EOL and has been removed from several
distributions.

Upstream-Status: Pending

Signed-off-by: Andrey Zhizhikin <andrey.z@gmail.com>
---
 scripts/team_basic_test.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/team_basic_test.py b/scripts/team_basic_test.py
index b05be9e..ad980e8 100755
--- a/scripts/team_basic_test.py
+++ b/scripts/team_basic_test.py
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 """
 Basic test.
 
-- 
2.17.1

